View/Converters/DrinkConverterOnboarding.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Controller;
+using View.Pages.Register;
+using Model;
+using System.Globalization;
+
+namespace View.Converters
+{
+    public class DrinkConverterOnboarding : IValueConverter
+    {
+        public static User user = Step1.User;
+        /// <summary>
+        /// Checks if a drink is already selected for favorite, like or dislike. If that is the case then it sends back true wich means it will show selected on the screen.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="targetType"></param>
+        /// <param name="parameter"></param>
+        /// <param name="culture"></param>
+        /// <returns></returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            string drinkName = (string)value;
+            string[] favorite = user.GetFavourites().Select(x => x.Name).ToArray();
+            string[] liked = user.GetLikes().Select(x => x.Name).ToArray();
+            string[] disliked = user.GetDislikes().Select(x => x.Name).ToArray();
+
+            if (!drinkName.Equals("False"))
```

Comment:
What the fuck gebeurt hier

Reply:
Deze functie zorgt ervoor dat wanneer de drankjes worden ingeladen de juiste drankjes geselecteerd worden met de juiste keuze.